RecoTracker/LSTCore/interface/HitsSoA.h

```diff
@@ -15,7 +15,7 @@ namespace lst {
                       SOA_COLUMN(uint16_t, moduleIndices),
                       SOA_COLUMN(unsigned int, idxs),
                       SOA_COLUMN(unsigned int, detid),
-                      SOA_COLUMN(unsigned int, clustSizes),
+                      SOA_COLUMN(size_t, clustSizes),
```

Comment:
64-bit type is too excessive here.
`uint8_t` should be enough.

Reply:
That makes sense to me.

For my knowledge, do you have an idea of where the type conversion should occur? e.g. should it occur as far away from the device as possible?

Reply:
It would be better to have the type settled before calling `LST::run`.

I missed that it was made `size_t` in the ntuple already.
Separately, it would be useful to switch that to a smaller type there (in `TrackingNtuple.cc`).
Perhaps it's better to switch to this smaller size type (`uint8_t`) there (ignoring a potential consistency argument to match the `pix_` and `str_` branches using cluster size).

Reply:
> It would be better to have the type settled before calling LST::run.

👍